### PR TITLE
Add a new top-level credits page

### PR DESCRIPTION
**Pull request by taldcroft**
This makes a new top-level Astropy Credits page including:
- Lifetime Contributors
- Institutional Support

This addresses https://github.com/astropy/astropy-project/issues/135 and the original discussion on astropy-dev "Proposal for giving credit for contributions" from May 2020.

The list of lifetime contributors was based on self-reported status in the last roles survey along with known "alumni" that are not currently active.

This does not include the credits listing for people who have made substantial contributions (at the 2,4,8,16 hrs per week level) in the last year that has been discussed previously. That is still planned but has been taken out of this PR to address some implementation concerns that came up in CoCo discussions.

Cc-ing folks that have participated in past discussions:
@eteq @mhvk @hamogu @astrofrog @kelle

### CODE REVIEW

**Comment by bsipocz** [comment]
Thank you @taldcroft.

I would suggest a read through for biases in the description and look for what type of verbs and statuses were used/highlighted.

**Comment by pllim** [comment]
Huh, what happened to the Giles HTML preview? Was it never set up?

**Comment by pllim** [comment]
@Cadair , is Giles broken?

https://github.com/astropy/astropy.github.com/blob/e276b7e81ff8bed7e9f85b83bc98448cba3efdb8/.circleci/config.yml#L32

File: credits.html [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+<h3>Chandra X-ray Center (CXC)</h3>
+The CXC has supported multiple staff members to work on the Astropy project,
+equivalent to   more than 5 person-years since the start of the project.
+
+<h3>NumFocus</h3>
+We wish to express gratitude to the NumFocus Organization for providing the
+organizational support to grow Astropy into the role of a community-leading
+project with a substantial budget.
+
+<h3>Center for Computational Astronomy (CCA), Flatiron Institute, Simons
```
Comment (mhvk) [inline credits.html:258]: Is this big enough to be mentioned? If so, I think the $30k grant from the Dunlap Institute should be mentioned too.

Though my sense would be to have a cut-off at ~1 person year.

File: credits.html [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+
+Here we recognize the institutions who have made major contributions to the
+Astropy project by either direct funding to the project or by indirect funding
+of employees who have contributed.
+
+<h3>Space Telescope Science Institute (STScI)</h3>
+STScI has played a foundational role in the development and advancement of the
+Astropy Project since its inception. STScI has provided continued and
+substantial support to the project via staff contributions since 2011, including
+X lifetime contributors to astropy.  Additionally, STScI has provided on-going
```
Comment (mhvk) [inline credits.html:239]: Don't forget to replace the `X` with an actual number...

File: credits.html [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+    Madison was a founding member of the Astropy project and was the main
+    developer of the io.fits core subpackage which forms the basis of much of
+    the I/O for astropy. After taking a hiatus from Astropy in 2016 for another
+    opportunity, Madison rejoined in the role of DevOps and Operations Support
+    in 2020.
+
+    <h3>Marten van Kerkwijk</h3>
+    Marten has been actively involved in astropy core development since 2013. He
+    has made many key contributions, most notably leading the effort to make
+    Quantities truly useful throughout astropy by making interaction with numpy
```
Comment (mhvk) [inline credits.html:157]: Maybe rephrase as "by working with and contributing to numpy to ensure interaction with numpy functions became seemless, and optimizing their use inside coordinates."

**Review by mhvk — COMMENTED** [review]
Good to have this. A few small comments only, perhaps the only bigger one being the level at which institutional contributions should start to count.

File: credits.html (hunk 1 above)
Comment (bsipocz) [reply credits.html:258]: And then DiRAC/eScience, too. And the Lorentz Centre probably, too.

**Comment by taldcroft** [comment]
> I would suggest a read through for biases in the description and look for what type of verbs and statuses were used/highlighted.

@bsipocz - good idea. I wrote most of these based on provided inputs and my own best effort. I'd be happy if an independent reviewer had a go at making the language neutral if there are issues.

**Comment by taldcroft** [comment]
All - about the other institutions, we discussed this in the CoCo previously and were generally agreed that support at the level of Dunlap, Lorentz etc should be mentioned. I just wanted to get this out for review to start the ball rolling. So please do suggest text!

File: credits.html [hunk 4]
```diff
@@ -0,0 +1,274 @@
...
+    role in making the Table class versatile.
+
+    <h3>Michael Droettboom</h3>
+    Michael was a prolific contributor to Astropy from 2011 through 2015,
+    contributing over 400 pull requests in many areas of the core, with a focus
+    on the wcs, votable, and table subpackages. His deep understanding of best
+    coding practices provided important inspiration for other members of the
+    initial core development team.
+
+    <h3>Moritz Guenther</h3>
```
Comment (hamogu) [inline credits.html:170]:
```suggestion
    <h3>Moritz Günther</h3>
```

**Comment by taldcroft** [comment]
Ping @lglatttly - any chance you'll be able to look at this?

File: credits.html (hunk 1 above)
Comment (adrn) [reply credits.html:258]: Agreed with @mhvk and @bsipocz! Also minor tweak:
```suggestion
<h3>Center for Computational Astrophysics (CCA), Flatiron Institute, Simons
```

**Comment by pllim** [comment]
If we near nothing from the copy-editor, is this good to go anyway?

**Comment by hamogu** [comment]
Unfortunately for us (but fortunately her her) our copy-editor has recently closed deal on two fictions books and won't be able to help out in astropy much in the foreseeable future. So, we'll have to do a closer review without her. @taldcroft can you address the review comments so we can read the text again in full before me merge? (It can always be improved later.)

File: credits.html [hunk 5]
```diff
@@ -0,0 +1,274 @@
...
+We wish to express gratitude to the NumFocus Organization for providing the
+organizational support to grow Astropy into the role of a community-leading
+project with a substantial budget.
+
+<h3>Center for Computational Astronomy (CCA), Flatiron Institute, Simons
+Foundation</h3>
+The CCA has provided logistical and travel support for the Python in Astronomy
+conference, the coordination meeting (dates), and the spectroscopy working
+group.
+
```
Comment (hamogu) [inline credits.html:263]: Add NASA since the grant is approved by now.

File: credits.html [hunk 6]
```diff
@@ -0,0 +1,274 @@
...
+    Adrian is the lead coordinator and developer behind the Learn Astropy and
+    Astropy tutorials initiatives. He was an initial contributor of the concept
+    and code for the Quantity class in astropy.units and serves as lead
+    developer of astropy.coordinates. In the latter role he played a key role in
+    adding support for velocity data such as proper motions in
+    astropy.coordinates.
+
+    <h3>Brigitta Sipőcz</h3>
+    Since coming to the project as a Google Summer of Code (GSoC) student in
+    2014, Brigitta has been an active and valued member of the Astropy community
```
Comment (hamogu) [inline credits.html:117]:
```suggestion
    Since coming to the project in
    2014, Brigitta has been an active and valued member of the Astropy community
```
We don't say that E.g. Eric joined the project as a student, either.

Comment (hamogu) [reply credits.html:117]: But thinking about it a little more, we might want to keep it in here and then add "s a student" to Eric, too, and say 'as a post-Doc' for Tom R and me. Not sure about the others, but it might be a good motivation for future, early career (e.g. student, Post-Doc) contributors!

**Comment by taldcroft** [comment]
FYI I'm on vacation this week.

File: credits.html [hunk 7]
```diff
@@ -0,0 +1,274 @@
...
+    in many key roles. Most notably, she has developed and maintains
+    responsibility for critical parts of the Astropy infrastructure which keep
+    the core package and affiliated packages running smoothly for both testing
+    and release distribution. She serves as an astropy core release manager and
+    serves as the GSoC coordinator for Astropy.
```
Comment (taldcroft) [inline credits.html:122]:
```suggestion
    and release distribution. She is an astropy core release manager and
    the GSoC coordinator for Astropy.
```

File: credits.html [hunk 8]
```diff
@@ -0,0 +1,274 @@
...
+
+<h3>NumFocus</h3>
+We wish to express gratitude to the NumFocus Organization for providing the
+organizational support to grow Astropy into the role of a community-leading
+project with a substantial budget.
+
+<h3>Center for Computational Astronomy (CCA), Flatiron Institute, Simons
+Foundation</h3>
+The CCA has provided logistical and travel support for the Python in Astronomy
+conference, the coordination meeting (dates), and the spectroscopy working
```
Comment (taldcroft) [inline credits.html:261]:
```suggestion
conference, the coordination meeting, and the spectroscopy working
```

File: credits.html [hunk 9]
```diff
@@ -0,0 +1,274 @@
...
+
+Here we recognize the institutions who have made major contributions to the
+Astropy project by either direct funding to the project or by indirect funding
+of employees who have contributed.
+
+<h3>Space Telescope Science Institute (STScI)</h3>
+STScI has played a foundational role in the development and advancement of the
+Astropy Project since its inception. STScI has provided continued and
+substantial support to the project via staff contributions since 2011, including
+X lifetime contributors to astropy.  Additionally, STScI has provided on-going
```
Comment (taldcroft) [inline credits.html:239]:
```suggestion
six lifetime contributors to astropy.  Additionally, STScI has provided on-going
```

File: credits.html [hunk 10]
```diff
@@ -0,0 +1,274 @@
...
+    developer of the io.fits core subpackage which forms the basis of much of
+    the I/O for astropy. After taking a hiatus from Astropy in 2016 for another
+    opportunity, Madison rejoined in the role of DevOps and Operations Support
+    in 2020.
+
+    <h3>Marten van Kerkwijk</h3>
+    Marten has been actively involved in astropy core development since 2013. He
+    has made many key contributions, most notably leading the effort to make
+    Quantities truly useful throughout astropy by making interaction with numpy
+    functions seamless and ensuring they work well inside coordinates. He led
```
Comment (taldcroft) [inline credits.html:158]:
```suggestion
    Quantities truly useful throughout astropy by working with and contributing to numpy to ensure interaction 
    with numpy functions became seemless, and optimizing their use inside coordinates. He led
```

File: credits.html (hunk 5 above)
Comment (taldcroft) [reply credits.html:263]: Do we actually have the money from the NASA grant yet and have spent some of it? Otherwise I think this can wait.

File: credits.html [hunk 11]
```diff
@@ -0,0 +1,274 @@
...
+    Astropy tutorials initiatives. He was an initial contributor of the concept
+    and code for the Quantity class in astropy.units and serves as lead
+    developer of astropy.coordinates. In the latter role he was a key player in
+    adding support for velocity data such as proper motions in
+    astropy.coordinates.
+
+    <h3>Brigitta Sipőcz</h3>
+    Since coming to the project in
+    2014, Brigitta has been an active and valued member of the Astropy community
+    in many key roles. Most notably, she has developed and maintains
```
Comment (taldcroft) [inline credits.html:118]:
```suggestion
    in many key roles. Most notably, she has been the lead developer and maintains
```

File: credits.html [hunk 12]
```diff
@@ -0,0 +1,274 @@
...
+    on the wcs, votable, and table subpackages. His deep understanding of best
+    coding practices provided important inspiration for other members of the
+    initial core development team.
+
+    <h3>Moritz Günther</h3>
+    Moritz has been involved in the Project since 2011, with contributions to
+    the io.ascii package and a continuing role as a package maintainer. He has
+    also contributed to the stats core subpackage and the photutils and saba
+    packages. Since 2020, Moritz has been serving as an Affiliated Package
+    review editor and has been an active member of the interim Finance Committee
```
Comment (hamogu) [inline credits.html:175]:
```suggestion
    review editor and has been an active member of the interim Finance Committee.
```

File: credits.html [hunk 13]
```diff
@@ -0,0 +1,274 @@
...
+
+<h3>Moore Foundation</h3>
+In late 2019 the Astropy project was awarded a major grant from the Gordon and
+Betty Moore Foundation. This grant was targeted at supporting Astropy’s
+transition to a fully sustainable project, where success no longer hinges on a
+limited set of contributors. This grant was transformative for the Project.
+
+<h3>Chandra X-ray Center (CXC)</h3>
+The CXC has supported multiple staff members to work on the Astropy project,
+equivalent to   more than 5 person-years since the start of the project.
```
Comment (hamogu) [inline credits.html:251]:
```suggestion
equivalent to more than 5 person-years since the start of the project.
```

File: credits.html (hunk 5 above)
Comment (hamogu) [reply credits.html:263]: I agree that the NASA grant can wait until money flows (soon, but not yet).

File: credits.html [hunk 14]
```diff
@@ -0,0 +1,274 @@
...
+    responsibility for critical parts of the Astropy infrastructure which keep
+    the core package and affiliated packages running smoothly for both testing
+    and release distribution. She is an astropy core release manager and
+    the GSoC coordinator for Astropy.
+
+    <h3>Erik Tollerud</h3>
+    Erik has played a key leadership and technical role in the Astropy project
+    since it began in 2011. His individual contributions are too numerous to
+    name, but they include contributing large parts of the core coordinates and
+    uncertainties packages, serving in the Coordination Committee since 2011,
```
Comment (taldcroft) [inline credits.html:128]:
```suggestion
    since it began in 2011. His contributions include writing large parts of the core coordinates and
    uncertainties packages, serving in the Coordination Committee since 2011,
```
